crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2398,6 +2465,10 @@ impl AuthorityPerEpochStore {
                         .or_default()
                         .push(tx.clone());
                 }
+                ConsensusCertificateResult::RandomnessConsensusMessage => {
```

Comment:
is this function guaranteed to be executed in the same order on all validators?
just making sure because `verify_consensus_transaction` above doesn't guarantee that

Reply:
I believe it is, @mystenmark can hopefully confirm

Reply:
The comment about verify_consensus_transactions being run in parallel is just wrong, as far as I can tell. I wonder if it used to be called by the batch verifier, which indeed does not ensure ordering.

In any case, handle_consensus_output implements a replicated state machine, so it should be executing identically on all validators